.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: go
 
 go:
-  - 1.5
```

Comment:
Its not the tests that fail, its the `install` step where we install `golint`.
I could write a custom install script that looks at the go version and then takes the decision whether or not to install `golint`
But, thats for another day.
I am hoping that `golint` craetes a tag/branch that works for Go 1.5, then I can put 1.5 back here in travis, add a custom install script to target that tag/branch